docs/tracing/tracing.md

```diff
@@ -0,0 +1,28 @@
+#### Tracing
+
+The cloudinfo application contains code instrumentation that makes possible tracing trough opencensus / jaegeR_
+
+For the time being the tracing instrumentation only covers background processes in the application (those collecting cloud information from the providers)
+
+#### Enable tracing
+
+The application comes with tracing disabled by default. Tracing can be configured through the following environment variables:
+
+| Envvar name                                | default value                                           |
+| -------------------------------------------| --------------------------------------------------------| 
+|  INSTRUMENTATION_JAEGER_ENABLED            | false                                                   |
+|  INSTRUMENTATION_JAEGER_COLLECTORENDPOINT  | http://localhost:14268/api/traces?format=jaeger.thrift  |
+|  INSTRUMENTATION_JAEGER_AGENTENDPOINT      | localhost:6832                                          |
+|  INSTRUMENTATION_JAEGER_SERVICENAME        | cludinfo                                                |
```

Comment:
This is not configurable, it's an alias pointing to a constant value. Aliases cannot be configured/overridden.